apps/mobile/src/libs/components/typing-text-view/typing-text-view.tsx

```diff
@@ -0,0 +1,47 @@
+import { Text } from "~/libs/components/components";
+import { BaseColor } from "~/libs/enums/enums";
+import { type ValueOf } from "~/libs/types/types";
+
+import { FIRST_LETTER_INDEX } from "./libs/constants/constants";
+import { TypingTextAnimation } from "./libs/enums/enums";
+import { useTypingText } from "./libs/hooks/hooks";
+
+type Properties = {
+	children: string;
```

Comment:
If it is just a string, then let's rename the prop. Children is for react components